### DISCUSSION TITLE
Question about additions & deletions

### DISCUSSION THREAD

**Discussion by ogradye** [community/community#22801] [New to GitHub]
<p>Hello, I’m curious to why certain lines on my commits show a deletion and an addition when I simply just commented something out. Didn’t backspace or anything. Shouldn’t this show as a modification and not a deletion or is it just the nature of Github to show as a deletion?</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="ogradye" data-post="1" data-topic="135593">
<div class="title">
<div class="quote-controls"></div>
 ogradye:</div>
<blockquote>
<p>Shouldn’t this show as a modification and not a deletion or is it just the nature of Github to show as a deletion?</p>
</blockquote>
</aside>
<p>It’s the nature of git in general to show changed lines as the old version being deleted and the new version added.</p>
